src/ccache.cpp

```diff
@@ -2273,6 +2275,7 @@ handle_main_options(int argc, const char* const* argv)
   static const struct option options[] = {
     {"cleanup", no_argument, nullptr, 'c'},
     {"clear", no_argument, nullptr, 'C'},
+    {"evict-older-than", required_argument, nullptr, 'e'},
     {"dump-manifest", required_argument, nullptr, DUMP_MANIFEST},
     {"dump-result", required_argument, nullptr, DUMP_RESULT},
```

Comment:
Keep the list sorted:

```suggestion
    {"dump-manifest", required_argument, nullptr, DUMP_MANIFEST},
    {"dump-result", required_argument, nullptr, DUMP_RESULT},
    {"evict-older-than", required_argument, nullptr, 'e'},
```